src/app/base/components/DebounceSearchBox/DebounceSearchBox.tsx

```diff
@@ -22,6 +22,8 @@ const DebounceSearchBox = ({
   onDebounced,
   searchText,
   setSearchText,
+  autoFocus,
+  placeholder,
```

Comment:
It looks like the type includes the `SearchBoxProps` but we don't spread them, which we could do instead of passing these specific props.